database/CorePartitionManager.java

```diff
@@ -125,19 +125,19 @@ protected Data(List<ColumnFamilyDescriptor> descriptors, List<ColumnFamilyHandle
                     configuration.defaultCFOptions()
```

Comment:
Note: the `DEFAULT` column family won't use the ID 0 when creating the column families, even though we'd like it to, because RocksDB requires it be called their built in default name